swagger.yaml

```diff
@@ -4100,6 +4100,8 @@ definitions:
       - logs-gather
       - next-step-runner
       - upgrade-agent
+      - download-boot-artifacts
+      - reboot
```

Comment:
Should we make this one more specific?  If I were to see this outside of the context of this PR, I would assume that it had to do with the reboot during installation.
If the instructions to the agent are generic in this step and will always remain generic, so that the step can be used for any scenario where we want to tell the agent to reboot, then leave it as "reboot".  If, on the other hand, the agent will perform some action specific to this step, I would call it something more specific.

Reply:
Good points! I think this instruction can be generic since it simply reboots the host and it can be reused in other scenarios too.

@carbonin what do you think?

Reply:
I guess the only thing that could be more specific is also running `chroot /host` before `reboot` on the agent. I'm not sure if `chroot /host` would be necessary to reboot the host in other scenarios

Reply:
Yeah, that bit is rather specific to our use case. I think it's probably safer to rename it to `reboot-for-reclaim` or something.